clusterctl/examples/google/machines.yaml.template

```diff
@@ -33,6 +35,8 @@ items:
       value:
         apiVersion: "gceproviderconfig/v1alpha1"
         kind: "GCEMachineProviderConfig"
+        roles:
+        - Node
```

Comment:
It seems like there should be a corresponding deletion on lines 49 and 50

Reply:
same as above.